src/Narochno.Primitives/project.json

```diff
@@ -8,11 +8,12 @@
     "projectUrl": "https://github.com/Narochno/Narochno.Primitives"
   },
 
-  "dependencies": {
-    "System.Collections": "4.3.0",
-    "System.Reflection.Extensions": "4.3.0",
-    "System.Reflection.TypeExtensions": "4.3.0"
-  },
+    "dependencies": {
+        "System.Collections": "4.3.0",
+        "System.Linq": "4.3.0",
+        "System.Reflection.Extensions": "4.3.0",
+        "System.Reflection.TypeExtensions": "4.3.0"

```

Comment:
It probably is.  We need to make a system for moving things like gitignore or editorconfig between repos.

Ideally, this would manage things like appveyor and/or Cake too.